src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2359,7 +2362,8 @@ private boolean isFeedbackSessionFullyCompletedByInstructor(
      */
     private boolean isFeedbackSessionViewableTo(
             FeedbackSessionAttributes session,
-            String userEmail) throws EntityDoesNotExistException {
+            String userEmail,
+            InstructorAttributes instructorOfCourse) throws EntityDoesNotExistException {
 
         if (session == null) {
```

Comment:
This check looks odd. I feel the method should not be receiving null sessions and therefore it should be an assertion instead.